.chloggen/lint-ci-jon-timeout-increased.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
```

Comment:
Please remove the changelog. You can prefix the PR with `[chore]` to skip the changelog check for a change like this 👍🏻